### PR TITLE
Disable cs2pr for now to see the actual output

### PR DESCRIPTION
**Pull request by staabm**

### CODE REVIEW

File: .github/workflows/static-analyze.yml [hunk 1]
```diff
@@ -126,4 +126,4 @@ jobs:
           cd ..
 
       - name: "analyze php code"
-        run: php -d memory_limit=-1 composer --working-dir=./.build psalm -- --shepherd --show-info=false --output-format=checkstyle | cs2pr
+        run: php -d memory_limit=-1 composer --working-dir=./.build psalm -- --shepherd --show-info=false --output-format=checkstyle
```
Comment (staabm) [inline .github/workflows/static-analyze.yml:123]: Psalm with activated shepherd will only work starting with the next release
https://github.com/vimeo/psalm/pull/2779

**Comment by prisis** [comment]
You need to remove the php check, else it only run on php files changes

**Comment by staabm** [comment]
Feel free to add the necessary changes

**Comment by prisis** [comment]
Found the error : 
`Run php -d memory_limit=-1 composer --working-dir=./.build phpstan -- --error-format=checkstyle`

`Could not open input file: composer`

Thanks for your help

**Pull request closed without merging**